build/release.sh

```diff
@@ -21,7 +21,8 @@ then
     git reset --hard origin/master
     git push --delete origin website || true
 
-    sbt -DscalaVersion=$SCALA_VERSION ++$SCALA_VERSION 'release with-defaults'
+    sbt -DscalaVersion=$SCALA_VERSION_2_12 ++$SCALA_VERSION_2_12 'release with-defaults'
+    sbt -DscalaVersion=$SCALA_VERSION_2_11 ++$SCALA_VERSION_2_11 'release with-defaults'
```

Comment:
first release 2.12 then 2.11, we got troubles with spark afaik and `-DscalaVersion` workaround